backend/services/automations/src/utils/actions/email.ts

```
@@ -1,3 +1,531 @@
+        toEmails = [...toEmails, ...emails];
+        continue;
+      }
+
+      if (type === 'customMail') {
+        const emails = config[key] || [];
+
+        toEmails = [...toEmails, ...emails];
+        continue;
+      }
+
+      if (!!reciepentType.pluginName) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)